### PR TITLE
Add reason in op governance vote

### PR DESCRIPTION
**Pull request by yabirgb**
Closes https://github.com/orgs/rotki/projects/11?pane=issue&itemId=51248759

### CODE REVIEW

File: rotkehlchen/chain/evm/decoding/interfaces.py [hunk 1]
```diff
@@ -169,16 +174,34 @@ def __init__(  # pylint: disable=super-init-not-called
...
             return DEFAULT_DECODING_OUTPUT
 
-        proposal_id = str(hex_or_bytes_to_int(context.tx_log.data[:32]))
-        supports = bool(hex_or_bytes_to_int(context.tx_log.data[32:64]))
-        notes = f'Voted {"FOR" if supports else "AGAINST"} {self.protocol} governance proposal {self.proposals_url}/{proposal_id}'  # noqa: E501
+        try:
+            _, decoded_data = decode_event_data_abi_str(context.tx_log, event_abi)
+        except DeserializationError as e:
+            log.error(
+                f'Failed to deserialize Juicebox event at '
```
Comment (LefterisJP) [inline rotkehlchen/chain/evm/decoding/interfaces.py:192]: Dude please don't copy pasta like this

```suggestion
                f'Failed to decode vote_cast event ABI at '
```

File: rotkehlchen/chain/evm/decoding/interfaces.py [hunk 2]
```diff
@@ -169,16 +174,34 @@ def __init__(  # pylint: disable=super-init-not-called
...
+                f'{context.transaction.tx_hash.hex()} due to {e}',
+            )
+            return DEFAULT_DECODING_OUTPUT
+
+        proposal_id = decoded_data[0]
+        supports = decoded_data[1]
+
```
Comment (LefterisJP) [inline rotkehlchen/chain/evm/decoding/interfaces.py:199]:
```suggestion
        proposal_id = decoded_data[0]
        supports = decoded_data[1]
```

File: rotkehlchen/chain/evm/decoding/interfaces.py [hunk 3]
```diff
@@ -169,16 +174,34 @@ def __init__(  # pylint: disable=super-init-not-called
...
+        notes_reason = ''
+        if len(decoded_data[3]) != 0:
```
Comment (LefterisJP) [inline rotkehlchen/chain/evm/decoding/interfaces.py:201]: had to use the ABI cause this is variable length and hard to do manually?

**Review by LefterisJP — COMMENTED** [review]
copy pasta + changelog

File: rotkehlchen/chain/evm/decoding/interfaces.py (hunk 3 above)
Comment (yabirgb) [reply rotkehlchen/chain/evm/decoding/interfaces.py:201]: yes, couldn't get it working manually

File: rotkehlchen/chain/evm/decoding/interfaces.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/chain/evm/decoding/interfaces.py:192]: I copied it and said: ok I need to change it but first let me see if it works fine... and then I forgot :(

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**